akka-persistence/src/test/scala/akka/persistence/PersistentActorSpec.scala

```diff
@@ -248,6 +248,14 @@ object PersistentActorSpec {
         persistAsync(event) { evt ⇒ sender() ! s"${evt.data}-b-${sendMsgCounter.incrementAndGet()}" }
     }
   }
+  class PersistAllNilPersistentActor(name: String) extends ExamplePersistentActor(name) {
+
+    val receiveCommand: Receive = commonBehavior orElse {
+      case Cmd(data) ⇒
+        persistAll(Nil)(_ ⇒ ())
```

Comment:
Hmm, persistAll invokes the handler once for every element—which means not at all in this case. I don’t think there is a need to loop anything here.